pythran/cxxgen.py

```diff
@@ -174,6 +173,17 @@ def get_tp():
             yield "} "
         return get_tp(), ""
 
+# literal values
+
+class IntegralConstant(object):
+    """Declare an integral constant known at compile time. Type and value are strings"""
+    def __init__(self, typename, value):
+        self.typename = typename
+        self.value = value
+
+    def __str__(self, with_semicolon=False):
```

Comment:
It's an expression, so it's probably useless to add this ``with_semicolon`` arg

Reply:
Yep, and it's also very useless because it's an argument to ``__str__``...